src/tinyvec.rs

```diff
@@ -26,6 +26,7 @@ use alloc::vec::{self, Vec};
 /// let many_ints: TinyVec<[i32; 4]> = tiny_vec!(1, 2, 3, 4, 5, 6, 7, 8, 9, 10);
 /// ```
 #[macro_export]
+#[cfg_attr(docs_rs, doc(cfg(target_feature = "alloc")))]
```

Comment:
😕 this is better in the short-form display not having `feature=""`, but is misleading in the long-form saying "This is supported with **target feature** `alloc` only."

I'd suggest using `feature = "alloc"` and making rustdoc render features better.

Reply:
(https://github.com/rust-lang/rust/pull/75330)

Reply:
oh that's just a copy/paste error.

i took it from safe_arch's intrinsics.